File: dayTwo/JAVASCRIPT/level_two/task_ten.js
const prompt = require("prompt-sync")();

let number = Number(prompt("Enter a number: "));

if (number % 3 == 0 && number % 5 == 0)
    console.log("It is divisible by 3 and 5");

else
    console.log("It is not divisible by 3 and 5");

File: dayTwo/JAVA/levelTwo/TaskTen.java
import java.util.Scanner;

public class TaskTen {

    public static void main(String... args) {

        Scanner inputCollector = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int number = inputCollector.nextInt();

        if (number % 5 == 0 && number % 3 == 0)
            System.out.println("It is divisible by both 5 and 3");
        else
            System.out.println("It is not divisible by both 5 and 3");
    }
}
